Runes.xcodeproj/project.pbxproj

```diff
@@ -713,6 +726,7 @@
 		51DE8A2B1BAB36E600124320 /* Release */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				APPLICATION_EXTENSION_API_ONLY = NO;
```

Comment:
This is for the test bundle. I'm actually setting this to `YES` at the project level so that _all_ of the frameworks have it set properly, but the tests need to have this set to `NO`. It's a confusing diff.